src/actions/user.actions.js

```diff
@@ -91,9 +92,23 @@ const update = (
   };
 };
 
+const resetPwd = (email) => {
+  return async (dispatch) => {
+    try {
+      await userService.resetPassword(email);
+      dispatch({ type: userConstants.USER_RESET_PASSWORD_EMAIL_SENT });
+      history.push(loginPageLink);
```

Comment:
Try to keep dummy the actions as much as possible and not use logic before the action is dispatched, this allows centralize the logic on the components and prevent side effects(i.e an action A has dispatched logout the user, this causes `history.push(loginPageLink)` is not worked as expected)

Reply:
And where do I should to execute that?